drivers/storage/portworx/component/pvccontroller.go

```diff
@@ -309,17 +309,14 @@ func (c *pvcController) createDeployment(
 		"--leader-elect=true",
 		"--controllers=persistentvolume-binder,persistentvolume-expander",
 		"--use-service-account-credentials=true",
+		"--leader-elect-resource-name=portworx-pvc-controller",
+		fmt.Sprintf("--leader-elect-resource-namespace=%s", cluster.Namespace),
 	}
 
-	if c.k8sVersion.LessThan(k8sutil.K8sVer1_24) {
-		if pxutil.IsOpenshift(cluster) {
-			command = append(command, "--leader-elect-resource-lock=endpoints")
-		} else {
-			command = append(command, "--leader-elect-resource-lock=configmaps")
-		}
+	if pxutil.IsOpenshift(cluster) {
+		command = append(command, "--leader-elect-resource-lock=endpoints")
```

Comment:
Is there benefit of using endpoints? Can't we always just use config maps now that we have the resource name fixed. I'm trying to keep the code simple and reduce branches.

Reply:
We should not use configmaps for new k8s versions. Those are deprecated in newer versions of k8s leader election. We should be using `leases`.
This is the reason we had the k8s version check because leases was added later and configmaps was the old way of doing leader election.